content/docs/5.0/get-started/tikv-in-5-minutes.md

```diff
@@ -0,0 +1,209 @@
+---
+title: TiKV in 5 Minutes
+description: Get started using TiKV in 5 Minutes
+menu:
+    "5.0":
+        parent: Get Started
+        weight: 1
+---
+
+## Prerequisites
+
+Ensure that you are using MacOS or Linux.
+
+{{< warning >}}
+The following steps are only for test environments.
+{{< /warning >}}
+<!-- TODO: For production environments please refer to ??? -->
+
+## Step1: Install a Playground TiKV Cluster
+
+1. Install TiUP by executing the following command:
+```bash
+$ curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh
```

Comment:
```suggestion
curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh
```

Reply:
so that the users can directly copy the whole command line.

Reply:
document in v4.0 and v3.0 contains `$`